_amenities/pools/swim-lessons.md

```diff
@@ -9,30 +9,32 @@ show-title: true
 ---
 
 <div class="alert alert-warning">
-<strong>NOTE:</strong> Klahanie pools are open and swim lessons are being offered with limitations. 
+  
+ Klahanie pools are open, and swim lessons are available. For the safety of our residents and staff, we follow CDC guidelines for the prevention of COVID-19. 
+ 
+<strong>Quick Links</strong> 
```

Comment:
What are you trying to accomplish with this change? This moves the Quick links header out of the floating card on the right side of the page and puts it into the yellow alert banner.